Documentation note about ES read-only (#3182)
chef/chef-server: docs-chef-io/content/server/ctl_chef_server.md

    option only works when run on a standalone Chef Infra Server or on a
    primary backend Chef server within a legacy tier.

{{< note >}}

> If the search does not work when data is present even after reindex, please verify the configurations of the search index. The following command will get the configuration of search index :

Reviewer: s/ If knife search does not seem to return the expected results and data is present in the chef-server database after reindex.

Perhaps also add an example of what missing looks like.

Reviewer 2: ```suggestion
If `knife search` does not return the expected results and data is present in the Chef Infra Server after reindex, then verify the search index configuration with the command:
```

>   ```bash

Reviewer 2: Add a blank line before and after each code block.

Author: have made the changes. Thank you.

  curl -XGET http://127.0.0.1:9200/_all/_settings`
  ```

> Make sure that the attribute `read_only_allow_delete` is not set to true. The following command will reset the configuration of search index :

Reviewer 2: ```suggestion
Set the `read_only_allow_delete` to false. Use this command to reset the search index configuration :

```

    option only works when run on a standalone Chef Infra Server or on a
    primary backend Chef server within a legacy tier.

{{< note >}}
If `knife search` does not return the expected results and data is present in the Chef Infra Server after reindex, then verify the search index configuration with the command:

>   ```bash
>   curl -XGET http://127.0.0.1:9200/_all/_settings`
>   ```

Reviewer 3: ````suggestion
```bash
curl -XGET http://127.0.0.1:9200/_all/_settings`
```
````

Set the `read_only_allow_delete` to false. Use this command to reset the search index configuration :

>   ```bash
>   curl -XPUT -H "Content-Type: application/json" http://127.0.0.1:9200/_all/_settings -d '{"index.blocks.read_only_allow_delete": null}'
>   ```

Reviewer 3: ````suggestion
```bash
curl -XPUT -H "Content-Type: application/json" http://127.0.0.1:9200/_all/_settings -d '{"index.blocks.read_only_allow_delete": null}'
```
````
